### PR TITLE
add: eslint for TS react.js library

### PR DESCRIPTION
**Pull request by sabesansathananthan**
Setup eslint for TS react.js library #3

nadunindunil/material-jsonschema-form#3

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,7 +21,11 @@
     "predeploy": "cd example && npm install && npm run build",
     "deploy": "gh-pages -d example/build"
   },
-  "dependencies": {},
+  "dependencies": {
+    "eslint-config-prettier": "^6.4.0",
+    "eslint-plugin-prettier": "^3.1.1",
+    "prettier": "^1.18.2"
+  },
```
Comment (nadunindunil) [inline package.json]: please add these libs into dev dependencies

**Review by nadunindunil — CHANGES_REQUESTED** [review]
please attend to requested

**Comment by sabesansathananthan** [comment]
done !

**Review by nadunindunil — APPROVED** [review]
thank you

**Pull request merged**